tests/data/linearityDet.yaml

```diff
@@ -0,0 +1,981 @@
+AMP:
+  C00:
+    FIT_PARAMS:
+    - 25087.503340648258
+    FIT_PARAMS_ERR:
+    - 0.0
+    FIT_RESIDUALS:
+    - 0.00016276041696983157
+    - -0.00032552083393966313
+    - 0.00016276041606033687
+    - .nan
```

Comment:
Also why are there nans here?

Reply:
These nans appear to be from the linearity code masking all of these values (the values are taken directly from the calibration).  I'd have to sit down and debug the linearity fits to figure out why.  https://jira.lsstcorp.org/browse/DM-42525